src/applications/claims-status/components/TabItem.jsx

```diff
@@ -14,24 +14,35 @@ class TabItem extends React.Component {
     document.removeEventListener('keydown', this.tabShortcut);
   }
 
+  // Grab the current URL, trim the leading '/', and return activeTabPath
+  trimCurrentUrl() {
```

Comment:
Does `trimCurrentUrl` need to be bound to `this`?

Reply:
@markgreenburg That's a good call, I do believe `trimCurrentUrl` does need to be bound. I'll refactor and retest before merging. Thank you.

Reply:
@markgreenburg I updated this as `trimCurrentUrl = () => { ... }`. Passes the tests, and worked locally.